src/navigation/BottomTabNavigator.tsx

```diff
@@ -1,5 +1,6 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Added this, so not all import and stuff related to repetition exercise needs to be comment out, as long as we do not publish this feature, but only the relevant lines need to be commented out. Added a hint to the clean up ticket, that this comment needs to be removed.